src/precomputed_quantities.jl

```diff
@@ -5,6 +5,23 @@
 import Thermodynamics as TD
 import ClimaCore.Fields as Fields
 import ClimaCore.Geometry as Geometry
+import ClimaCore.Spaces:
+    column, ExtrudedFiniteDifferenceSpace, FiniteDifferenceSpace
+
+Base.@propagate_inbounds function column(
```

Comment:
This has already been added to the latest version of ClimaCore?

Reply:
Yes it has. Removed from PR in latest commit